### DISCUSSION TITLE
Drawer component transitionDuration prop is not working

### DISCUSSION THREAD

**Discussion by ghost** [palantir/blueprint#5872] [Q&A] [answered]
#### Environment

- Package version: 3.49.1
- Operating System: Mac OS catalina
- Browser name and version: electron (google chrome)
This is my component:

            <Drawer
                onClose={() => setIsDrawerOpen(false)}
                title={"Some title" }
                size={DrawerSize.LARGE}
                isOpen={isOpen}// a state that changes after pressing a button
                shouldReturnFocusOnClose={true}
                position={Position.RIGHT}
                transitionDuration={10000}//NOTHING CHANGES!!!!!!!!!
                style={style}
            >

#### Steps to reproduce

Create a drawer with transitionDuration={10000} and see nothing changes with the duration of the transition

#### Actual behavior

nothing changes with the duration of the transition of the drawer

#### Expected behavior

Drawer transition duration to open and close in 10 seconds

**Comment by adidahiya** [comment, marked as answer]
In order to customize transition duration (generally not recommended, as it's part of the design system), you need to also override styles which define the CSS transition. We use a custom mixin for this; you'll have to find the relevant class name and add custom styles on top of it:

https://github.com/palantir/blueprint/blob/88a158b38a96350109650a7fe821a5091a25b3ed/packages/core/src/components/drawer/_drawer.scss#L30
